python/rikai/types/vision.py

```diff
@@ -67,9 +69,9 @@ def __init__(
     def from_array(
         cls,
         array: np.ndarray,
-        uri: Union[str, Path],
-        mode: str = None,
-        format: str = None,
+        uri: Optional[Union[str, Path]] = None,
```

Comment:
now that we have an embedded image version should we separate out the saving to uri portion as a separate method call rather than optional parameters?

Reply:
like `Image.as_external(uri)` or sth?

Reply:
we can do in a follow up JIRA?

Reply:
#249